cloudinit/config/cc_ca_certs.py

```diff
@@ -44,60 +44,110 @@
 from cloudinit import subp
 from cloudinit import util
 
-CA_CERT_PATH = "/usr/share/ca-certificates/"
-CA_CERT_FILENAME = "cloud-init-ca-certs.crt"
-CA_CERT_CONFIG = "/etc/ca-certificates.conf"
-CA_CERT_SYSTEM_PATH = "/etc/ssl/certs/"
-CA_CERT_FULL_PATH = os.path.join(CA_CERT_PATH, CA_CERT_FILENAME)
-
-distros = ['alpine', 'debian', 'ubuntu']
+DISTRO_DEFAULT_CONFIG = {
```

Comment:
I'm not sure Ubuntu/Debian/Alpine is set as `DEFAULT_CONFIG`, because I don't know cloud-init's default distros..
However, if we can use I agree your proposal

Reply:
cloud-init doesn't really have default distros. All I mean here is that those three distros share a common config, so we only need to store it once; as that'll be what we use if there isn't a more specific configuration to use, it is the default configuration, by definition.

Does that make sense?